n-puzzle-solver.py

```diff
@@ -70,19 +78,30 @@ def get_no_of_misplaced_tiles_count(puzzle_board):
                 count = count + 1
     return count
 
-def best_first_search_euclidean_distance(puzzle_board):
+def get_astar_manhattan_distance_average_steps_position():
+          return 0, 0
+def get_astar_euclidean_distance_average_steps_position():
+    return 0, 1
+def get_astar_misplaced_tiles_average_steps_position():
+    return 0, 2
+
+def astar_algorithm(puzzle_board, heuristic_function_type, get_average_steps_position):
     # Solving puzzle using Best first search Euclidean distance heuristics

```

Comment:
can you correct this comment line. "Solving puzzle using A start algorithm" would be appropriate.